lms/djangoapps/course_home_api/outline/views.py

```diff
@@ -275,14 +276,14 @@ def get(self, request, *args, **kwargs):  # pylint: disable=too-many-statements
             if allow_public or user_is_masquerading:
                 handouts_html = get_course_info_section(request, request.user, course, 'handouts')
 
-        if not show_enrolled:
+        if not is_enrolled:
```

Comment:
This is the actual one-line fix.

`show_enrolled = is_enrolled || is_staff`

But even staff can't enroll manually in invitation-only style courses. So we just need to look at the appropriate variable here.